tests/cpgrid/addLgrsOnDistributedGrid_test.cpp

```diff
@@ -409,21 +398,6 @@ void refinePatch_and_check(Dune::CpGrid& coarse_grid,
     BOOST_CHECK( static_cast<int>(allGlobalIds_cells.size()) == global_cells_count);
     BOOST_CHECK( allGlobalIds_cells.size() == allGlobalIds_cells_set.size() );
 
-    // Check global id is not duplicated for interior points
-    std::vector<int> localInteriorPointIds_vec;
-    localInteriorPointIds_vec.reserve(data.back()->size(3)); // more than actually needed since only care about interior points
-    int local_interior_points_count = 0;
-    for (const auto& point: vertices(coarse_grid.leafGridView(), Dune::Partitions::interior)) {
-        localInteriorPointIds_vec.push_back(data.back()->globalIdSet().id(point));
-        ++local_interior_points_count;
-    } 
-    auto global_point_count  = coarse_grid.comm().sum(local_interior_points_count);
-    auto [allGlobalIds_points, displ_point] = Opm::allGatherv(localInteriorPointIds_vec, coarse_grid.comm());
-
-    const std::set<int> allGlobalIds_points_set(allGlobalIds_points.begin(), allGlobalIds_points.end());
-    BOOST_CHECK( static_cast<int>(allGlobalIds_points.size()) == global_point_count);
-    BOOST_CHECK( allGlobalIds_points.size() == allGlobalIds_points_set.size() );
```

Comment:
This is deactivated because there is a bug right? Please add a prominent comment and create a github issue about this.